docs/book/src/component-config-tutorial/testdata/project/config/rbac/kustomization.yaml

```diff
@@ -16,3 +16,9 @@ resources:
 - auth_proxy_role.yaml
 - auth_proxy_role_binding.yaml
 - auth_proxy_client_clusterrole.yaml
+# For each CRD, "Editor" and "Viewer" roles are scaffolded by
+# default, aiding admins in cluster management. While optional
+# for managers, who can modify or remove them, their removal
+# means they won't be installed with your solution.
```

Comment:
@lunarwhite 

Sorry, now I check the comment here better.
See that Manager means the "cmd/main.go" -> The Project Itself
We call Manager because it Manages Controllers
So, i think we need to improve this comment here.